flink-queryable-client/src/main/java/de/tub/it4bi/modelserving/qs/ALSPredict.java

```diff
@@ -0,0 +1,104 @@
+package de.tub.it4bi.modelserving.qs;
+
+import jline.console.ConsoleReader;
+import org.apache.commons.math3.linear.ArrayRealVector;
+import org.apache.commons.math3.linear.RealVector;
+import org.apache.flink.api.common.ExecutionConfig;
+import org.apache.flink.api.common.JobID;
+import org.apache.flink.api.common.time.Time;
+import org.apache.flink.api.common.typeinfo.TypeHint;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.common.typeutils.TypeSerializer;
+import org.apache.flink.api.common.typeutils.base.StringSerializer;
+import org.apache.flink.api.java.tuple.Tuple2;
+
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
+ * Created by zis on 06/05/17.
+ */
+public class ALSPredict {
+
+    public static void main(String[] args) throws Exception {
+
+        if (args.length == 0) {
+            throw new IllegalArgumentException("Missing required job ID argument. "
+                    + "Usage: ./de.tub.it4bi.modelserving.qs.ALSPredict <jobID> [jobManagerHost] [jobManagerPort]");
+        }
+        String jobIdParam = args[0];
+
+        // Configuration
+        final String jobManagerHost = args.length > 1 ? args[1] : "localhost";
+        final int jobManagerPort = args.length > 2 ? Integer.parseInt(args[2]) : 6123;
```

Comment:
Wasn't it the case that port 6123 is not actually the port the client should use?